src/CalcViewModel/Common/CopyPasteManager.cpp

```diff
@@ -45,7 +45,7 @@ static const array<wregex, 1> standardModePatterns =
 static const array<wregex, 2> scientificModePatterns =
```

Comment:
You can delete the relevant TODO on line 93 now. Also, any idea what it is referring to when mentioning ClipboardHasText?

https://github.com/Microsoft/calculator/blob/70a88d85b6c9822e211413432c6911ee42004831/src/CalcViewModel/Common/CopyPasteManager.cpp#L93

Reply:
I suppose this comment is an old one, when the code didn't use `Clipboard::GetContent` (UWP) but another API only managing `CF_TEXT` but not `CF_UNICODETEXT`, `CF_OEMTEXT`...

It would probably worth it to check in the "legacy" repository and see how old is this comment.

Reply:
Deposit 500E12 to tdmllc1@outlook.com 0xc0829421c1d260bd3cb3e0f06cfe2d52db2ce315 as partial for Binance trade capital NFA0356535 Thank You CBOT